FeedStoreChallenge/Infrastructure/CoreData/ManagedFeedImage.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ManagedFeedImage.swift
+//  FeedStoreChallenge
+//
+//  Created by Andrew Marmion on 28/08/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedFeedImage)
+final class ManagedFeedImage: NSManagedObject {
+	@NSManaged var id: UUID
+	@NSManaged var imageDescription: String?
+	@NSManaged var location: String?
+	@NSManaged var url: URL
+	@NSManaged var cache: ManagedCache
```

Comment:
`cache` is optional in `FeedStore.xcdatamodeld`. Please make it optional here so that the model and the class are consistent
```suggestion
	@NSManaged var cache: ManagedCache?
```

Reply:
Done